pytest_jupyterhub/jupyterhub_spawners.py

```diff
@@ -5,9 +5,18 @@
 
 import pytest
 from jupyterhub.spawner import Spawner
+from traitlets.config import Config
 
 
 @pytest.fixture
 def spawner_class():
     """Allows tests to setup the Spawner class that will be used by the hub."""
     return Spawner
+
+
+@pytest.fixture
+def jupyterhub_spawner_config(spawner_class):
```

Comment:
Let's be more explicit that we're configuring only the hub's `spawner_class` through this and not the `spawner` object
```suggestion
def jupyterhub_spawner_class_config(spawner_class):
```